containers/base/javascript/browser/css/dashboard.css

```diff
@@ -56,3 +163,670 @@ progress[value]::-moz-progress-bar {
 	font-family: Lato;
 	font-size: 13px;
 }
```

Comment:
Spotted some tabs here. I'll merge this PR and replace the tabs with spaces. Just FYI.